README.md

```diff
@@ -48,6 +48,42 @@ xmlns:xct="http://xamarin.com/schemas/2020/toolkit">
 
 The documentation is still under construction, but we've published the most important things over at [Microsoft Docs](https://docs.microsoft.com/en-us/xamarin/community-toolkit/). If you want to contribute some of the missing bits you can do this over at the [official Docs repo](https://github.com/MicrosoftDocs/xamarin-communitytoolkit) as contributions are very much welcomed!
 
+## DrawingView
```

Comment:
Let's not add these to the README, this should go into the Docs. Remove it from here, I have copied it so we can add it to the Docs when this gets merged